src/Container/Config.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace WPML\PB\Container;
+
+class Config {
+
+	public static function getSharedClasses() {
+		return [
+			'\WPML_PB_Factory',
+			'\WPML_PB_Integration',
```

Comment:
This `\WPML_PB_Integration` class has a lot of code in it and it's getting hard to extend and test properly. Having a unique instance of this class allows to use is as a dependency in `WPML\PB\AutoUpdate\Hooks`.